runtime/mainnet/src/config/assets.rs

```diff
@@ -0,0 +1,583 @@
+					.unwrap_or_default();
+
+			let max_collection_account_size =
+				pallet_nfts::CollectionAccount::<Runtime>::storage_info()
+					.first()
+					.and_then(|info| info.max_size)
+					.unwrap_or_default();
+
+			let total_collection_size = max_collection_size +
+				max_collection_role_size +
+				max_collection_config_size +
+				max_collection_account_size;
+			println!("{:?}", deposit(4, total_collection_size));
+			// 4 different storage items means 4 different keys.
+			assert_eq!(deposit(4, total_collection_size), NftsCollectionDeposit::get());
+		}
+
+		#[test]
+		fn collection_id_is_u32() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::CollectionId>(),
+				TypeId::of::<u32>(),
+			);
+		}
+
+		#[test]
+		fn create_origin_ensures_signed() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::CreateOrigin>(),
+				TypeId::of::<AsEnsureOriginWithArg<EnsureSigned<AccountId>>>(),
+			);
+		}
+
+		#[test]
+		fn balances_provides_currency() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::Currency>(),
+				TypeId::of::<Balances>(),
+			);
+		}
+
+		#[test]
+		fn ensure_deposit_per_byte_deposit() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::DepositPerByte>(),
+				TypeId::of::<NftsDepositPerByte>(),
+			);
+			assert_eq!(
+				<<Runtime as pallet_nfts::Config>::DepositPerByte as Get<Balance>>::get(),
+				deposit(0, 1)
+			);
+		}
+
+		#[test]
+		fn all_feature_are_active() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::Features>(),
+				TypeId::of::<NftsPalletFeatures>(),
+			);
+
+			let features = [Trading, Attributes, Approvals, Swaps];
+
+			for feat in features {
+				assert!(<Runtime as pallet_nfts::Config>::Features::get().is_enabled(feat));
+			}
+		}
+
+		#[test]
+		fn force_origin_ensures_root() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::ForceOrigin>(),
+				TypeId::of::<EnsureRoot<AccountId>>(),
+			);
+		}
+
+		#[cfg(feature = "runtime-benchmarks")]
+		#[test]
+		fn benchmark_helper_is_default() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::Helper>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn max_attributes_per_item_is_30() {
+			assert_eq!(
+				<<Runtime as pallet_nfts::Config>::ItemAttributesApprovalsLimit as Get<u32>>::get(),
+				30
+			);
+		}
+
+		#[test]
+		fn ensure_item_deposit_deposit() {
+			let max_size = pallet_nfts::Item::<Runtime>::storage_info()
+				.first()
+				.and_then(|info| info.max_size)
+				.unwrap_or_default();
+			assert_eq!(deposit(1, max_size), NftsItemDeposit::get());
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::ItemDeposit>(),
+				TypeId::of::<NftsItemDeposit>(),
+			);
+		}
+
+		#[test]
+		fn item_id_is_u32() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_nfts::Config>::ItemId>(),
+				TypeId::of::<u32>(),
+			);
+		}
+
+		#[test]
+		fn attribute_key_maximum_lenght_is_64() {
```

Comment:
typo in length

Reply:
[84188ad](https://github.com/r0gue-io/pop-node/pull/465/commits/84188ad70ecf80bf071da0098dbe66ffc5649e49)